apis/v1alpha1/generator.yaml

```diff
@@ -179,14 +189,6 @@ resources:
         from:
           operation: RestoreDBInstanceFromDBSnapshot
           path: UseDefaultProcessorFeatures
-    renames:
-      operations:
-        CreateDBInstance:
-          input_fields:
-            EnablePerformanceInsights: PerformanceInsightsEnabled
-        ModifyDBInstance:
-          input_fields:
-            EnablePerformanceInsights: PerformanceInsightsEnabled
```

Comment:
This change is still necessary. Even if it doesn't break functionality, we want to reduce duplication between fields in the spec and status. Since this property of `DBInstance` is configurable (it isn't read-only), it should appear only in the spec. We should do the same for the other properties we identified as well.